instrumentation/jms-1.1/javaagent/src/jms2Test/groovy/SpringTemplateJms2Test.groovy

```diff
@@ -112,7 +112,6 @@ class SpringTemplateJms2Test extends AgentInstrumentationSpecification {
         session -> template.getMessageConverter().toMessage("responded!", session)
       }
     }
-    // wait for thread to start, we expect the first span to be from receive
```

Comment:
I believe this comment was added when we tried to control the order of operation with a `CountDownLatch`. Later `CountDownLatch` was replaced with sorting traces but this comment wasn't removed.